third_party/proto/cosmos_proto/cosmos.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package cosmos_proto;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/regen-network/cosmos-proto";
+
+extend google.protobuf.MessageOptions {
+    string interface_type = 93001;
+
+    string implements_interface = 93002;
+}
+
+extend google.protobuf.FieldOptions {
+    string accepts_interface = 93001;
+}
```

Comment:
did these come from somewhere? where are they defined?

Reply:
third party dependencies for the other protobuf files